react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -178,11 +184,16 @@ class ConnectionStatsTable extends Component<Props> {
     render() {
         const { isLocalVideo, enableSaveLogs, disableShowMoreStats } = this.props;
         const className = isMobileBrowser() ? 'connection-info connection-info__mobile' : 'connection-info';
+        const connectionInfoStyles = {
+            maxHeight: this.props._filmstripHeight - 14,
```

Comment:
I'm wondering if something like `calc(100%-14)` would work? This may reduce the rerenders when we change the filmstrip height.

Reply:
Tested out your suggestion and it doesn't work.